github/interactions_repos.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// GetInteractions fetches the interaction restrictions for a repository.
+//
+// GitHub API docs: https://developer.github.com/v3/interactions/repos/#get-interaction-restrictions-for-a-repository
+func (s *InteractionsService) GetInteractions(ctx context.Context, owner string, repo string) (*Interaction, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/interaction-limits", owner, repo)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeRepositoryInteractionsPreview)
+
+	repositoryInteractions := new(Interaction)
+
+	resp, err := s.client.Do(ctx, req, repositoryInteractions)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return repositoryInteractions, resp, nil
+}
+
+// UpdateInteractions adds or updates the interaction restrictions for a repository.
+//
+// GitHub API docs: https://developer.github.com/v3/interactions/repos/#add-or-update-interaction-restrictions-for-a-repository
+func (s *InteractionsService) UpdateInteractions(ctx context.Context, owner string, repo string, interaction *Interaction) (*Interaction, *Response, error) {
```

Comment:
Here, and on the other functions as well, let's take advantage of Go's parameter type shorthand:

s/owner string, repo string/owner, repo string/

Reply:
I'm aware of the shorthand notation in case all the parameters are of the same type.

But I didn't get this below line.
> s/owner string, repo string/owner, repo string/

Reply:
Addressed in the new commit - https://github.com/google/go-github/pull/1079/commits/a8446a5e03d2118bccaf382d31d5921ef3d58708